Базовая стилизация всех страниц

OliaErmak opened the pull request:
---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/33/user/315091/tasks/5)

:boom: https://htmlacademy-htmlcss.github.io/315091-sedona-33/2/

---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/33/user/315091/tasks/6)

OliaErmak committed:
Добавляет разметку главной и каталога

OliaErmak committed:
Добавила графику для всех страниц

Экспортировала и оптимизировала изображения

OliaErmak committed:
Подключает контентные изображения к разметке

OliaErmak committed:
Добавляет width и height в некоторых местах

OliaErmak committed:
Удаляет лишнюю строку

ad0v0 commented:
**Проверка разметки главной и каталога:**

1) Проверка задания согласно критериям:
- ✅ [PROJ-01. Кодгайд](https://up.htmlacademy.ru/htmlcss/32/criteries#PROJ-01) - ок 
- ☑️  [PROJ-02. Именование](https://up.htmlacademy.ru/htmlcss/32/criteries#PROJ-02):
-- hotel-form слишком подвязанный под свой контент и расположение класс. Как насчет класса `intro`? а внутри для блока с формой можно сделать обертку с классом `form-filter-wrapper`:
```
section.intro
  h1
  ol с хлебными крошками
  div.form-filter-wrapper
    form
```
- ✅ [PROJ-03. Название папок и файлов](https://up.htmlacademy.ru/htmlcss/32/criteries#PROJ-03) - ок 
- ✅ [HTML-01. Ориентиры](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-01) - ок
- ☑️  [HTML-02. Заголовки](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-02):
-- http://joxi.ru/gmvXlEbtezoPRA нет заголовка первого уровня на странице с каталогом
-- http://joxi.ru/52apBKvtlnz7Mm если секция вложена в секцию, то заголовок во вложенной секции должен иметь меньший увовень
- ✅ [HTML-03. Кнопки и ссылки](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-03):
- ✅ [HTML-04. Работоспособные формы](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-04) - ок
- ✅ [HTML-05. Подписи полей форм](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-05) - ок
- ☑️  [HTML-07. Валидация HTML](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-07) - есть ошибка на странице с каталогом:
-- http://joxi.ru/Rmzyo6bCvlePZr 
- ✅ [IMG-01. Контентная и декоративная графика](https://up.htmlacademy.ru/htmlcss/32/criteries#IMG-01) - ок 
- ✅ [CONT-01. Текстовые переносы](https://up.htmlacademy.ru/htmlcss/32/criteries#CONT-01) - ок 
- ✅ [ALLY-01. Описание графики](https://up.htmlacademy.ru/htmlcss/32/criteries#ALLY-01) - ок 
- ☑️  [ALLY-02. Подписи интерактивных элементов](https://up.htmlacademy.ru/htmlcss/32/criteries#ALLY-02):
-- http://joxi.ru/krDQRKjIG6jepm описать словами , что должно произойти при клике на кнопку . Касается и минуса
2) Разное:
- ☑️  http://joxi.ru/J2bzBZDHqy6972 это декоративные иконки, как кнопки эти календари не функционируют, для инициации появления выбора даты будет выступать все поле

🎌 - есть замечание, надо править
➡️ - есть комментарий, желательно учесть
✅ - выполнено 
☑️ - правка по замечанию внесена
❓ - есть вопрос, желательно ответить

ПС Ошибки , которые дублируются в коде, в замечаниях не дублирую, но перепроверять и править нужно везде и обязательно

ad0v0 commented:
**Проверка графики и базовой стилизации главной и каталога:**

1) Проверка работы согласно критериям:
- ✅ [IMG-02. Форматы графики](https://up.htmlacademy.ru/htmlcss/32/criteries#IMG-02) - ок 
- ✅ [IMG-03. Размеры графики в разметке](https://up.htmlacademy.ru/htmlcss/32/criteries#IMG-03) - ок
- [PROJ-01. Кодгайд](https://up.htmlacademy.ru/htmlcss/32/criteries#PROJ-01)
- [PROJ-02. Именование](https://up.htmlacademy.ru/htmlcss/32/criteries#PROJ-02)
- [PROJ-04. Закомментированный код](https://up.htmlacademy.ru/htmlcss/32/criteries#PROJ-04)
- [PROJ-05. Неиспользуемые файлы](https://up.htmlacademy.ru/htmlcss/32/criteries#PROJ-05)
- [TEST-05. Шрифты](https://up.htmlacademy.ru/htmlcss/32/criteries#TEST-05)
- [TEST-06. Pixel Perfect](https://up.htmlacademy.ru/htmlcss/32/criteries#TEST-06)
- ☑️ [ALLY-01. Описание графики](https://up.htmlacademy.ru/htmlcss/32/criteries#ALLY-01):
-- http://joxi.ru/8Anlb8pCNVJ34m в конце каждого содержимого alt ставим точку. См критерий https://up.htmlacademy.ru/htmlcss/33/criteries#ALLY-01 
- [CSS-01. Подключение стилей](https://up.htmlacademy.ru/htmlcss/32/criteries#CSS-01)
- [CSS-02. Глобальный селектор](https://up.htmlacademy.ru/htmlcss/32/criteries#CSS-02)
- [CSS-03. Вложенность](https://up.htmlacademy.ru/htmlcss/32/criteries#CSS-03)
- [CSS-04. ID в селекторах](https://up.htmlacademy.ru/htmlcss/32/criteries#CSS-04)
- [CSS-05. Использование !important](https://up.htmlacademy.ru/htmlcss/32/criteries#CSS-05)
- [CSS-06. Использование шрифтов](https://up.htmlacademy.ru/htmlcss/32/criteries#CSS-06)
2) Разное:

🎌 - есть замечание, надо править
➡️ - есть комментарий, желательно учесть
✅ - выполнено 
☑️ - правка по замечанию внесена
❓ - есть вопрос, желательно ответить

ПС Ошибки , которые дублируются в коде, в замечаниях не дублирую, но перепроверять и править нужно везде и обязательно

ad0v0 commented:
стилизацию в этот же пулл реквест добавляем

ad0v0 commented:
По графике и разметке проверка готова 👌🏻  жду правки и стилизацию

OliaErmak committed:
Исправляет замечания по разметке

Главная:
1 Изменила заголовок вложенной секции с h2 на h3
2 Ошибка в валидации - добавила  action=""
3 Изменила описание кнопок - и + в форме
4 Изменила с button.calendar на div.calendar
5 Добавила точки в конце alt

Каталог:
1 Изменила именование с hotel-form на intro, добавила обертку для формы div.form-filter-wrapper
2 Добавила точки в конце alt
3 Модифицировала блок причин

OliaErmak committed:
Переименовывает контентные картинки на главной

ad0v0 commented:
- http://joxi.ru/Dr8YPw4SKWK8pm нет заголовка второго уровня

OliaErmak committed:
Добавляет заголовок

Добавила заголовок h2

OliaErmak committed:
Добавила базовую стилизацию

OliaErmak committed:
Добавила стилизацию формы в каталоге

ad0v0 commented on styles/styles.css:
```diff
@@ -0,0 +1,330 @@
+}
+
+
+
```
форматирование кода: оставляем в конце файла одну пустую строку, остальные лишние

ad0v0 commented on styles/styles.css:
```diff
@@ -0,0 +1,330 @@
+  color: #000000;
+  background-color:#ffffff;
+
+  background-image: none;
```
форматирование: поправить группировку css свойств внутри правила - фоновые свойства не должны быть разьединены пустой строкой

ad0v0 commented on styles/styles.css:
```diff
@@ -0,0 +1,330 @@
+.sort-by,
+.numbers-hotels {
+  font-size: 18px;
+  line-height: 23.31px;
```
округляем значения  согласно правилам математики или в сторону большего, дробные могут по разнмоу обрабатываться разными браузерами (какие-то могут самостоятельно округлять значения)

ad0v0 commented on styles/styles.css:
```diff
@@ -0,0 +1,330 @@
+
+/* убрала стили у филдсет */
+
+fieldset {
```
https://up.htmlacademy.ru/htmlcss/33/criteries#CSS-02 использовать класс . Например, reset-fieldset

ad0v0 commented on styles/styles.css:
```diff
@@ -0,0 +1,330 @@
+.form-filter-wrapper label,
+.reset,
+.sort-by,
+.numbers-hotels {
```
рекомендую не группировать в одно правило разные компоненты, тк таким образом код усложняется в поддержке, поскольку усложняется его логика

ad0v0 commented on styles/styles.css:
```diff
@@ -0,0 +1,330 @@
+  background-image: url("../images/index-background.jpg");
+  background-repeat: no-repeat;
+  background-size: 100% auto;
+  text-align: center; /* надо ли тут центровать так? */
```
поскольку мы не центрируем еткстовый контент, я все-таки рекомендовала бы использовать центрирование, например, с помощью margin: 0 auto

ad0v0 commented:
- http://joxi.ru/E2pZpJBCvyj4Xr размер шрифта
- http://joxi.ru/krDQRKjIGXdgXm стилизовать текст в формах
- http://joxi.ru/YmE53Z9SGxLGLm перепроверить и поправить стилизацию
- http://joxi.ru/a2XNn0KiQWpQqr шрифт в селекте ниже тоже
